ui/pages/create-account/import-account/bottom-buttons.js

```diff
@@ -0,0 +1,48 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import {
+  ButtonPrimary,
+  ButtonSecondary,
+  BUTTON_SECONDARY_SIZES,
+} from '../../../components/component-library';
+import Box from '../../../components/ui/box/box';
+import { DISPLAY } from '../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import * as actions from '../../../store/actions';
+
+BottomButtons.propTypes = {
+  importAccountFunc: PropTypes.func.isRequired,
+  isPrimaryDisabled: PropTypes.bool.isRequired,
+};
+
+export default function BottomButtons({
+  importAccountFunc,
+  isPrimaryDisabled,
+}) {
+  const t = useI18nContext();
+  const dispatch = useDispatch();
+
+  return (
+    <Box display={DISPLAY.FLEX}>
```

Comment:
Need to add a gap here
```suggestion
    <Box display={DISPLAY.FLEX} gap={4}>
```
<img width="593" alt="Screenshot 2023-03-02 at 1 29 40 PM" src="https://user-images.githubusercontent.com/8112138/222559284-fe263136-3a98-432e-bbe9-a69b4f780138.png">